src/main/plugins/org.dita.base/xsl/preprocess/topicpullImpl.xsl

```diff
@@ -691,8 +691,12 @@ mode="topicpull:figure-linktext" and mode="topicpull:table-linktext"
   <!-- Get the short description for a link or xref -->
   <xsl:template match="*" mode="topicpull:get-stuff_get-shortdesc">
     <xsl:param name="targetElement" as="element()?"/>
+    <xsl:param name="in-desc-content" as="xs:boolean" select="false()" tunnel="yes"/>
     
     <xsl:choose>
+      <!-- if already creating content inside a <desc>, do not create a lower-level <desc>
+           (see #4184 for details on how this can occur) -->
```

Comment:
The comment should not reference a bug report. Instead describe what's going on without pointing to old problems.